tests/smoke-tests/test-randomness-consistency.ts

```diff
@@ -343,14 +343,26 @@ function isRandom(bytes: Uint8Array) {
 
 // Tests if byte output is independent
 function chiSquareTest(bytes: Uint8Array) {
-  let chiSquared = 0;
-  // expected value is expected average of [u8; 32]
-  const expectedValue = 128;
-  // degrees of freedom is 32 - 1 = 31, alpha is 0.05
-  // chi.pdf(31, 0.05) = 44.985
+  let chiSquared = 0.0;
+  let numOnes = 0;
+  // expected value of 256 coin flips:
+  const expectedValue = 256 / 2;
+  // degrees of freedom is 256 - 1 = 255, alpha is 0.05
+  // chi.pdf(31, 0.05) = 287.882 (TODO: use precise value; this is from 250 in following table)
```

Comment:
```suggestion
  // chi.pdf(250, 0.05) = 287.882 (TODO: use precise value; this is from 250 in following table)
```

So from this estimated pValue = 287.882 - 1 ?

Reply:
It's the value for `250`, the nearest-neighbor (or that was the intent). I imagine we can find a precise value somewhere.